frontend/js/views/list-annotation.js

```diff
@@ -352,9 +362,16 @@ define(["jquery",
                 modelJSON.duration     = (modelJSON.duration || 0.0);
                 modelJSON.textHeight   = this.$el.find("span.freetext").height();
 
-                if (modelJSON.isMine && this.scale && modelJSON.label.category.scale_id) {
+                if (modelJSON.label) {
                     category = annotationTool.video.get("categories").get(this.model.get("label").category.id);
 
+                    title = modelJSON.label.abbreviation + " - " + modelJSON.label.value;
+                    this.$el.css("background-color", category.get("settings").color);
```

Comment:
Does not handle undefined categories, which can for example happen if the category of the label was deleted.

Reply:
Oof, shoot. Why is that even allowed?!

Reply:
OK, I guess for now we have to deal with that. We should think hard about whether that's something we want, though. Maybe ... some time in the future ... A man can dream. Until then, I created #436.

I will update this shortly.